<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlayerUI</class>
 <widget class="QWidget" name="PlayerUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>842</width>
    <height>491</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PlayerUI</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#PlayerUI
{
	
	background-color: rgb(53, 53, 53);
}

QLabel
{
	
	color: rgb(255, 255, 255);
	font: 10pt &quot;黑体&quot;;
}</string>
  </property>
  <widget class="VideoWidget" name="srcwidget1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>380</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBoxplay">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>381</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>播放模块</string>
   </property>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>50</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_stop">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>40</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>暂停</string>
    </property>
   </widget>
   <widget class="QSlider" name="verticalSlider">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>40</y>
      <width>21</width>
      <height>41</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_openvideo">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>打开</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_export">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>50</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>导出</string>
    </property>
   </widget>
  </widget>
  <widget class="QSlider" name="playerslider">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>361</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>999</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="VideoWidget" name="deswidget">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>10</y>
     <width>380</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBoxset">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>290</y>
     <width>401</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>设置模块</string>
   </property>
   <widget class="QPushButton" name="btn_set">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>140</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>设置</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelbright">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>亮度</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spinbox_bright">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <double>99.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spinbox_contrast">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>60</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <double>1.000000000000000</double>
    </property>
    <property name="maximum">
     <double>3.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="labelcontrast">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>对比度</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelrotate">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>旋转</string>
    </property>
   </widget>
   <widget class="QComboBox" name="combo_rotate">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>100</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>90</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>180</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>270</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="combo_flip">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>140</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>不处理</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>上下</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>左右</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>上下左右</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="labelflip">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>镜像</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelsize">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>20</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>尺寸</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_w">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>20</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_h">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>20</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelsize_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>60</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>图像金字塔</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_gaosi">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>60</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_laplase">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>60</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_word">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>100</y>
      <width>41</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>水印</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_wordw">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>100</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_wordh">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>100</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_worda">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>100</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QSlider" name="playerslider_begin">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>361</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>999</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="playerslider_end">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>361</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>999</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <zorder>groupBoxset</zorder>
  <zorder>srcwidget1</zorder>
  <zorder>groupBoxplay</zorder>
  <zorder>playerslider</zorder>
  <zorder>deswidget</zorder>
  <zorder>playerslider_begin</zorder>
  <zorder>playerslider_end</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>VideoWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>videowidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
